Formula/r/ronn-ng.rb

```diff
@@ -0,0 +1,36 @@
+class RonnNg < Formula
+  desc "Build man pages from Markdown"
+  homepage "https://github.com/apjanke/ronn-ng"
+  url "https://github.com/apjanke/ronn-ng/archive/refs/tags/v0.10.1.tar.gz"
+  sha256 "180f18015ce01be1d10c24e13414134363d56f9efb741fda460358bb67d96684"
+  license "MIT"
+
+  # Nokogiri 1.9 requires a newer Ruby
+  depends_on "ruby"
+
+  conflicts_with "ronn", because: "ronn also ships a ronn binary"
+
```

Comment:
we can do a followup on this
```suggestion
```

Reply:
just remove the line, we can do the conflicts_with PR as follows.

Reply:
(the way the conflicts_with works, it requires both of the two formulae have it, but it would trigger a rebottle for ronn if we do it)